### DISCUSSION TITLE
Autocompletion with Parallel

### DISCUSSION THREAD

**Discussion by SeSa99** [eviden-parallel/community#23] [Q&A]
Hello, is there a possibility to autoimport / autocomplete Parallel components?

**Comment by Th3S4mur41** [comment]
The core package relies entirely on semantic HTML and proper aria attribute.
So as long as you write correct and accessible HTML, styles will be applied automatically.
You'll only need to apply additional classes for special cases and modifiers (e.g. `accent-2` to apply the accent color instead of the default one).

Components are written with [Lit](https://lit.dev/), so if you are working in [VS Code](https://code.visualstudio.com/), you could give one of those plugins a try:
- [lit-plugin](https://marketplace.visualstudio.com/items?itemName=runem.lit-plugin)
- [lit-html](https://marketplace.visualstudio.com/items?itemName=bierner.lit-html)
